.github/workflows/push.yml

```diff
@@ -22,6 +26,14 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: '3.x'
+    - name: Cache tools
+      id: cache-linux
+      uses: actions/cache@v2
+      with:
+        path: |
+          ./tools/dist
+          /home/runner/arduino_ide
```

Comment:
Is it possible to use some environment variable instead of `/home/runner`?

Reply:
This is what I was referring to in the description.  I couldn't find an env variable for this.

Reply:
I tried $HOME,it returns the home directory (i.e. /home).
I think what can be done here is to export the variable from our script as a Github Action output.

Reply:
Oh, right, i understand that point now!
It seems that this action supports `~` (tilde) expansion, https://github.com/actions/cache/issues/664#issuecomment-1004662677.

Reply:
Tilde expansion seems to work.  I tried `$HOME` before but that didn't work.